README.md

```diff
@@ -231,8 +166,6 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
 ## ⭐️ Show your support <a name="support"></a>
 
 > Write a message to encourage readers to support your project
```

Comment:
- [ ] Kindly remove this section description because it is just an instruction for you to make your readme file once you make it then these instructions are no more useful 🤔
- [ ] Also, it would be great if you add a good message in this section such as **If you like this project then don't forget to give star ⭐ to this repository** to make your readme file more descriptive 😊